pubsub/kafka/kafka.go

```diff
@@ -123,7 +127,12 @@ func (k *Kafka) Init(metadata pubsub.Metadata) error {
 
 	k.brokers = meta.Brokers
 	k.producer = p
-	k.consumerGroup = meta.ConsumerID
+	if meta.ConsumerID != "" {
```

Comment:
It feels like you could handle this case in `getKafkaMetadata` and could remove this given my comment below.

Reply:
done